drud/ddev
# Update requirements.txt for mkdocs changes, fixes #3291

## The Problem/Issue/Bug:

Update mkdocs-material and related packages for #3249:
* Update mkdocs-material and dependencies
* Logic fix to fix-search.js
* Put in jquery before putting in fix-search.js so it can use jquery 

## Manual Testing Instructions:

Visit a docs page as described in #3291 and check js console in browser. Shouldn't see complaints.



<a href="https://gitpod.io/#https://github.com/drud/ddev/pull/3294"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
